typescript/types/change-case/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@labelflow/types-change-case",
+  "repository": "git@github.com:labelflow/labelflow.git",
+  "version": "0.0.1",
+  "types": "src/index.d.ts",
+  "peerDependencies": {
+    "change-case": "^4.1.2",
+    "type-fest": "^2.12.0"
```

Comment:
Ahah! Nice to see that you've noted it 😉 
The ` @labelflow/types-change-case` package I've added in this PR is just like a `@types/*` package. It doesn't adds any actual code and is only based on 2 packages whose API are stable-enough to be used with `^`. It comes in conjunction with using `peerDependencies` so that the version pinning responsibility is delegated to the higher-level packages.

In our current context it's not that useful as the dependency management between our projects is kind of hacky but at the end it should minimize the need to maintain minor versions and revisions on our types-* packages.